packages/background/package.json

```diff
@@ -3,20 +3,22 @@
   "version": "1.0.0",
   "main": "index.js",
   "license": "MIT",
-	"private": true,
+  "private": true,
   "devDependencies": {
     "@babel/cli": "^7.16.0",
     "@babel/core": "^7.16.0",
     "@babel/preset-env": "^7.16.4",
     "@babel/preset-typescript": "^7.16.0",
     "@types/chrome": "^0.0.164",
+    "@types/firefox-webext-browser": "^94.0.1",
     "@typescript-eslint/eslint-plugin": "^5.4.0",
     "@typescript-eslint/parser": "^5.4.0",
     "babel-loader": "^8.2.3",
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.25.3",
     "typescript": "^4.5.2",
+    "webextension-polyfill": "^0.8.0",
```

Comment:
maybe add this into the global package.json together with firefox-webext-browser